### ISSUE TITLE
Deprecations

### ISSUE THREAD

**Issue by thirtythree** [Simbul/baker#1020]
I'm new to Baker and have just run into an issue where when I run the project in the simulator i see all these errors come up within xcode.

Does any one know if these are bad?

thanks in advance for your help.

![screen shot 2013-07-16 at 2 23 57 pm](https://f.cloud.github.com/assets/1758685/804572/d7ac0bd6-ee12-11e2-9b04-895472b4a6c1.png)

**Comment by folletto** [comment]
Some random points:
1. If the app complies and runs, it's ok. Corollary: warnings aren't an issue, just read them and understand them.
2. What version of Baker are you using?
3. What version of Xcode are you using?
4. What version of iOS are you using?

**Comment by thirtythree** [comment]
Thanks for the response.

To answer your questions, 
2. I'm using baker 4.1
3. Im using xcode 4.5.1
4. im currently running Lion

Are these all ok to use?

**Comment by folletto** [comment]
Yep but: which iOS version? 6.1?

**Comment by folletto** [comment]
Also: upgrade to the latest version of Xcode ;)

**Comment by thirtythree** [comment]
the ios version is 6.0

**Comment by folletto** [comment]
I just noticed from another issue (#993) that the deprecations come out when you change the build iOS target. Did you change the target to 6.0?

**Comment by thirtythree** [comment]
Yes I did should it be an older one

**Comment by folletto** [comment]
Well, that was kinda the question I was going to ask to you. :)
Baker has a default setting, and that's what we support. If you change it... then you need to take care of the impact of such changes. :)

So: Why did you change it?

**Comment by folletto** [comment]
Hello, do you have any update? :)
Why did you change the iOS target version? :)

**Comment by thirtythree** [comment]
I'm sorry I thought I'd responded to that.

My changing of the iOS target was an oversight on my part their was know reason.

I have got it working now and am waiting on the ios dev centre to come back on line so I can start the process of getting it approved for the newsstand.

Thanks so much for your help.
